src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.tsx

```diff
@@ -196,47 +266,53 @@ const FixCommitmentInfo: React.FC<Props> = ({
                   </Typography>
                 </Box>
               </Grid>
-
-              <Grid item xs={12}>
-                <Box>
-                  {data.contacts.nodes.map((contact) => (
-                    <Contact
-                      id={contact.id}
-                      name={contact.name}
-                      key={contact.name}
-                      statusTitle={
-                        contact.status
-                          ? contactPartnershipStatus[contact.status]
-                          : ''
-                      }
-                      statusValue={contact.status || ''}
-                      amount={contact.pledgeAmount || 0}
-                      amountCurrency={contact.pledgeCurrency || ''}
-                      frequencyTitle={
-                        contact.pledgeFrequency
-                          ? frequencies[contact.pledgeFrequency]
-                          : ''
-                      }
-                      frequencyValue={contact.pledgeFrequency || ''}
-                      hideFunction={hideContact}
-                      updateFunction={updateContact}
-                      statuses={contactStatuses || [{ name: '', value: '' }]}
-                      setContactFocus={setContactFocus}
-                    />
-                  ))}
-                </Box>
-              </Grid>
               <Grid item xs={12}>
-                <Box className={classes.footer}>
-                  <Typography>
-                    <Trans
-                      defaults="Showing <bold>{{value}}</bold> of <bold>{{value}}</bold>"
-                      shouldUnescape
-                      values={{ value: data.contacts.nodes.length }}
-                      components={{ bold: <strong /> }}
-                    />
-                  </Typography>
-                </Box>
+                <InfiniteList
```

Comment:
Could you make the same changes to InfiniteList that you did in the other PR so the hover goes away?